recipe/meta.yaml

```diff
@@ -38,13 +37,13 @@ requirements:
     # in the Python modules.
     - arrow-cpp {{ version }}
     - boost-cpp 1.82
-    - cython 0.29
+    - cython 0.29.36
```

Comment:
hmm,  what about strict pinning ? cython is C/C++ package, I have tried  >=0.29.36 ( what upstream requires ) but it failed for py 312, see https://cloud.prefect.io/anaconda/flow-run/f98b33ee-e839-4c34-87ce-347f109c0013?logs

Reply:
Cython is a Python package. It generates C++ code. And I agree a hard pin should be avoided. If something fails for Python 3.12, then we should investigate.

Reply:
It's best to avoid strict pinning for cython because it has good backward compatibility (newer cython version will generate code compatible with older python versions). Your log seems to indicate a pinning to 0.29.32 (which doesn't support 3.12).